docker/wine.sh

```diff
@@ -14,9 +14,12 @@ main() {
     # add repository for latest wine version and install from source
     # hardcode version, since we might want to avoid a version later.
     wget -nc https://dl.winehq.org/wine-builds/winehq.key
-    mv winehq.key /usr/share/keyrings/winehq-archive.key
+    # workaround for wine server synchronization, see #1035
+    mkdir -p /etc/apt/keyrings
+    mv winehq.key /etc/apt/keyrings/winehq-archive.key
     wget -nc https://dl.winehq.org/wine-builds/ubuntu/dists/focal/winehq-focal.sources
     mv winehq-focal.sources /etc/apt/sources.list.d/
+    sed -i s@/usr/share/keyrings/@/etc/apt/keyrings/@ /etc/apt/sources.list.d/winehq-focal.sources || true
```

Comment:
can you section this with some newlines to make it clear what the workaround is

Reply:
Does that look better?

Reply:
nice!